desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphNodeBuilder.java

```diff
@@ -302,7 +286,7 @@ private void addNodes(DirectedGraph<Object, JobGraphLink> graph, SourceColumnFin
                     final Table table = inputColumn.getPhysicalColumn().getTable();
                     if (table != null) {
                         addNodes(graph, scf, table, recurseCount);
-                        addEdge(graph, table, item, null, null);
+                        addEdge(graph, table, item, null, null, null);
```

Comment:
All these calls adding a null are a little nasty. Wouldn't an overloaded addEdge() be better?